src/authorization/Authorizations.ts

```diff
@@ -712,7 +744,12 @@ export default class Authorizations {
       // Access Control Enabled?
       if (!chargingStation.siteArea.accessControl) {
         // No ACL: Always try to get the user
-        return UserStorage.getUserByTagId(tenantID, tagID);
+        let user = await UserStorage.getUserByTagId(tenantID, tagID);
+        if (!user && Utils.isTenantComponentActive(tenant, TenantComponents.OICP)) {
```

Comment:
-> Call Hubject to know if thee badge ID exists in their backend

I thought about to authorize at this point by requesting Hubject. But it does not work because

- OICP authorize request automatically starts a session on Hubject 
- isTagIDAuthorizedOnChargingStation() method is called quite often

I decided to authorize the user when swipes the badge on the charging station.
in OCPPService->handleAuthorize()

Reply:
I don't know if it is a problem when I start a lot of session just to authorize.

Reply:
Yep but you cannot consider when the user is not found that you will always return the OICP one, that's not correct.
So user is unknown and this case has to be treated differently in the backend.
You can ask Hubject also how to solve this kind of issue.

Reply:
I asked them

Reply:
It should be no problem as long as the number of request is not too high.
Here is the response:

> Hi Jakob,
> 
> Since you are still in the QA/development phase it is alright if you test the AuthorizeStart/Stop using the RFID cards.
> That's the whole reason a CPO package was sent to you.
> 
> If the Sessions will stay open, it will not affect Hubject functionality in anyway because we have a session timeout duration > and eventually the Sessions will be closed.
> 
> Just be careful that you don't overload the system by sending thousands of request per day.
> 
> I am sure that will not be the case, but just a heads up.
> 
> Does this answer your questions?
> 
> 
> Best Regards,
> Mrudula Sathe

Reply:
I changed it

Reply:
@Jakob8 Did you push your changes?
I still see the getUserByEmail here instead of requesting an Auth:
![image](https://user-images.githubusercontent.com/25266945/105682246-80be8c00-5ef2-11eb-82d2-04422c49ce52.png)
Moreover, we must have an API at Hubject side to check this and which is valid for the production system.

Reply:
I wondering why you cannot see my changes

Reply:
![image](https://user-images.githubusercontent.com/51398311/105699701-13b5f100-5f08-11eb-9f75-abe5c31f6350.png)
I'm checking if the unknown user is a roaming user.

2 remarks:
1. I'm not throwing a BackendError inside authorizeStart because we want to check if user is a roaming user and if not the user is unauthorized and the process should continue -> create an inactive Tag in db
2.  I'm also checking for a roaming user in case the Tag in the db is inactive. That's the case if the user tried once to authorize and it was not successful, e.g. the OICP component was not active at this time or an error on Hubject occurred unauthorized is returned -> an inactive Tag is created in db.
Then this roaming user wants to authorize at a later point and has an inactive Tag in the db -> authorize over Hubject